### PR TITLE
BrenKt-SK951-WEB-Hosting-TUTO-create_website_start10m 

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web/hosting/create_your_personal_webpage/guide.es-es.md [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+Si no está familiarizado con el código en los diferentes lenguajes citados, puede copiar y pegar los ejemplos de código que figuran en esta guía, lo que le permitirá tener un sitio web explotable en su alojamiento.
+
+### ¿Qué herramientas utilizar?
+
+Para crear una página web, comience escribiendo en un archivo su código fuente a partir de uno de los tres lenguajes anteriores. Estos son sus principales nombres de extensión: *".html"* (para los archivos HTML), *".css"* (para los archivos CSS), *".js"* (para los archivos JavaScript).
+
+Los archivos pueden escribirse en editores de texto, incluyendo los que están disponibles por defecto en su sistema operativo (bloc de notas, TextEdit). Muchas soluciones de código abierto gratuitas ofrecen funcionalidades adicionales: [Notepad++](https://notepad-plus-plus.org/), [Brackets](https://brackets.io/), [Sublime Text](https://www.sublimetext.com/) o [micrófono](https://micro-editor.github.io/). También es posible utilizar un IDE (Integrated Development Environment, entorno de desarrollo integrado) como [Visual Studio Code](https://code.visualstudio.com/) o [Geany](https://www.geany.org/).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.es-es.md]:
```suggestion
Los archivos pueden escribirse en editores de texto, incluyendo los que están disponibles por defecto en su sistema operativo (bloc de notas, TextEdit). Muchas soluciones de código abierto gratuitas ofrecen funcionalidades adicionales: [Notepad++](https://notepad-plus-plus.org/), [Brackets](https://brackets.io/), [Sublime Text](https://www.sublimetext.com/) o [Micro](https://micro-editor.github.io/). También es posible utilizar un IDE (Integrated Development Environment, entorno de desarrollo integrado) como [Visual Studio Code](https://code.visualstudio.com/) o [Geany](https://www.geany.org/).
```

File: pages/web/hosting/create_your_personal_webpage/guide.es-es.md [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+
+#### Añadir imágenes para hacer su página más atractiva
+
+La web es un medio sobre todo visual. Veremos en esta sección cómo insertar imágenes en su contenido. El producto Start 10M le ofrece un espacio de almacenamiento de 10 MB. Esto es suficiente para las páginas HTML y CSS, pero puede ser limitado si desea poner muchas imágenes en su sitio web. En ese caso, le recomendamos que contrate un [plan de hosting de OVHcloud](https://www.ovhcloud.com/es-es/web-hosting/) que le permita disfrutar de un mayor almacenamiento.
+
+La etiqueta HTML usada para mostrar una imagen es la etiqueta `<img>`. A diferencia de las etiquetas que hemos visto antes, no hay apertura ni cierre de este elemento. Hablaremos de una etiqueta de autocierre. Los atributos de esta etiqueta son los que le permitirán vincular la ubicación del archivo con el texto descriptivo de la imagen.
+
+##### **Optimizar las imágenes**
+
+Una imagen de gran tamaño es una imagen que tardará en ser cargada por su navegador, especialmente si sus visitantes utilizan un smartphone o tablet conectada a la red 4 o 5G.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.es-es.md]:
```suggestion
Una imagen de gran tamaño es una imagen que tardará en ser cargada por su navegador, especialmente si sus visitantes utilizan un smartphone o tablet conectada a la red 4G o 5G.
```

File: pages/web/hosting/create_your_personal_webpage/guide.es-es.md [hunk 3]
```diff
@@ -0,0 +1,449 @@
...
+Para ganar tiempo en la apariencia de su sitio web y permitirle concentrarse en el contenido, y por lo tanto lo que será referenciado, es habitual recurrir a *templates* (modelos) para ganar tiempo y tener un resultado de calidad tanto gráfica como funcionalmente (diseño, usabilidad, visibilidad en smartphone y tablet).
+
+#### ¿Qué es una plantilla? ¿Qué soluciones utilizar?
+
+Una *plantilla* es un modelo o ejemplo que se puede reutilizar, adaptándolo o no. El recurso a las *plantillas* permite ganar tiempo en el diseño de un sitio adaptando elementos ya diseñados, al tiempo que ofrecen las cualidades que se pueden exigir de un sitio "profesional". También puede utilizarse la palabra "tema".
+
+Existen soluciones "open source" gratuitas disponibles en Internet, como [Bootstrap](https://materializecss.com/), [Materialize](https://materializecss.com/), [Foundation](https://get.foundation/) o incluso [Semantic UI](https://semantic-ui.com/). Estas herramientas se denominan "framework": se trata de librerías que facilitan la creación de sitios o aplicaciones web. Ofrecen elementos estandarizados, personalizables y reutilizables y toda la comunidad propone *templates* reutilizables.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.es-es.md]:
```suggestion
Existen soluciones "open source" gratuitas disponibles en Internet, como [Bootstrap](https://materializecss.com/), [Materialize](https://materializecss.com/), [Foundation](https://get.foundation/) o [Semantic UI](https://semantic-ui.com/). Estas herramientas se denominan "framework": se trata de librerías que facilitan la creación de sitios o aplicaciones web. Ofrecen elementos estandarizados, personalizables y reutilizables y toda la comunidad propone *templates* reutilizables.
```

File: pages/web/hosting/create_your_personal_webpage/guide.es-es.md [hunk 4]
```diff
@@ -0,0 +1,449 @@
...
+- [Guía de referencia de etiquetas HTML](https://developer.mozilla.org/es/docs/Web/HTML)
+- [Cursillo W3Schools sobre HTML](https://www.w3schools.com/html/)
+- [Tutoriels CSS Mozilla](https://developer.mozilla.org/es/docs/Web/CSS/Tutorials)
+- [CSS Tutorial W3 Schools](https://www.w3schools.com/css/).
+
+### Volver a trabajar sus imágenes
+
+Gracias a las numerosas herramientas gratuitas, podrá reelaborar sus ilustraciones:
+
+- La aplicación [Fotos](https://apps.microsoft.com/store/detail/photos-microsoft/9WZDNCRFJBH4) Windows 10 y 11
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.es-es.md]:
```suggestion
- La aplicación [Photos](https://apps.microsoft.com/store/detail/photos-microsoft/9WZDNCRFJBH4) Windows 10 y 11
```

File: pages/web/hosting/create_your_personal_webpage/guide.es-es.md [hunk 5]
```diff
@@ -0,0 +1,449 @@
...
+- [https://bootstrapthemes.co/](https://bootstrapthemes.co/).
+
+## Más información
+
+En internet encontrará muchos recursos para aprender y mejorar su práctica, copiar elementos enteros de código o fragmentos de código, añadir funcionalidades a su sitio web sin correr el riesgo de cometer errores o fallos de funcionamiento. A continuación, algunos de los sitios de referencia:
+
+- [Empezar con el HTML](https://developer.mozilla.org/es/docs/Learn/HTML/Introduction_to_HTML/Getting_started)
+- [Guía de referencia de etiquetas HTML](https://developer.mozilla.org/es/docs/Web/HTML)
+- [Cursillo W3Schools sobre HTML](https://www.w3schools.com/html/)
+- [Tutoriels CSS Mozilla](https://developer.mozilla.org/es/docs/Web/CSS/Tutorials)
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.es-es.md]:
```suggestion
- [Tutoriales CSS Mozilla](https://developer.mozilla.org/es/docs/Web/CSS/Tutorials)
```

File: pages/web/hosting/create_your_personal_webpage/guide.es-es.md [hunk 6]
```diff
@@ -0,0 +1,449 @@
...
+- La aplicación [Photos](https://support.apple.com/es-es/guide/photos/welcome/mac) macOS
+- [Paint.Net](https://www.getpaint.net/), [GIMP](https://www.gimp.org/), [darktable](https://www.darktable.org/)
+- Piense también en las aplicaciones de retoque de imagen disponibles en sus smartphones Android o iOS.
+
+También encontrará recursos en línea:
+
+- [Compresor](https://compressor.io/)
+- [ShrinkMe](https://shrinkme.app/)
+- [Free Online Image Optimizer](https://kraken.io/web-interface)
+- [TinyJPG](https://tinyjpg.com/) et [TinyPNG](https://tinypng.com/).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.es-es.md]:
```suggestion
- [TinyJPG](https://tinyjpg.com/) y [TinyPNG](https://tinypng.com/).
```

File: pages/web/hosting/create_your_personal_webpage/guide.it-it.md [hunk 7]
```diff
@@ -0,0 +1,449 @@
...
+**Ultimo aggiornamento: 22/11/2022**
+
+## Obiettivo
+
+Questa guida ti mostra come creare la prima pagina di un sito Web su un hosting Start 10M offerto gratuitamente con l'acquisto di un dominio OVHcloud.
+
+## Prerequisiti
+
+- Disporre di un [dominio](https://www.ovhcloud.com/it/domains/)
+- Disporre di un piano di [hosting Web OVHcloud](https://www.ovh.com/it/hebergement-web/) o di un [hosting gratuito Start 10M](https://www.ovhcloud.com/it/domains/free-web-hosting/)
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.it-it.md]:
```suggestion
- Disporre di un piano di [hosting Web OVHcloud](https://www.ovhcloud.com/it/web-hosting/) o di un [hosting gratuito Start 10M](https://www.ovhcloud.com/it/domains/free-web-hosting/)
```

File: pages/web/hosting/create_your_personal_webpage/guide.it-it.md [hunk 8]
```diff
@@ -0,0 +1,449 @@
...
+Se non conosci il codice nei diversi linguaggi citati, puoi copiare/incollare gli esempi di codice forniti in questa guida, ti permetteranno di avere un sito web utilizzabile sul tuo hosting.
+
+### Quali strumenti utilizzare?
+
+Per creare una pagina Web, inizia scrivendo in un file il tuo codice sorgente a partire da uno dei tre linguaggi citati sopra. Ecco i nomi principali dell'estensione: *".html"* (per i tuoi file HTML), *".css"* (per i tuoi file CSS), *".js"* (per i tuoi file JavaScript).
+
+I file possono essere scritti in editor di testo, inclusi quelli disponibili di default sul tuo sistema operativo (block note, TextEdit). Numerose soluzioni Open Source gratuite offrono funzionalità aggiuntive: [Notepad++](https://notepad-plus-plus.org/), [Bracket](https://brackets.io/), [Sublime Text](https://www.sublimetext.com/) o ancora [micro](https://micro-editor.github.io/). È inoltre possibile utilizzare un IDE (Integrated Development Environment, ambiente di sviluppo integrato) come [Visual Studio Code](https://code.visualstudio.com/) o [Geany](https://www.geany.org/).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.it-it.md]:
```suggestion
I file possono essere scritti in editor di testo, inclusi quelli disponibili di default sul tuo sistema operativo (block note, TextEdit). Numerose soluzioni Open Source gratuite offrono funzionalità aggiuntive: [Notepad++](https://notepad-plus-plus.org/), [Bracket](https://brackets.io/), [Sublime Text](https://www.sublimetext.com/) o ancora [Micro](https://micro-editor.github.io/). È inoltre possibile utilizzare un IDE (Integrated Development Environment, ambiente di sviluppo integrato) come [Visual Studio Code](https://code.visualstudio.com/) o [Geany](https://www.geany.org/).
```

File: pages/web/hosting/create_your_personal_webpage/guide.it-it.md [hunk 9]
```diff
@@ -0,0 +1,449 @@
...
+Puoi copiare/incollare questo codice nel tuo file `index.html`:
+
+```html
+<!DOCTYPE html>
+<html lang="it">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Ma page personnelle</title>
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.it-it.md]: This FR content should be translated

File: pages/web/hosting/create_your_personal_webpage/guide.it-it.md [hunk 10]
```diff
@@ -0,0 +1,449 @@
...
+
+#### Aggiungi immagini per rendere la tua pagina più attraente
+
+Il web è soprattutto un mezzo visivo. In questa sezione vedremo come inserire le immagini nel tuo contenuto. L'offerta Start 10M ti offre uno spazio di storage da 10 MB. È sufficiente per le tue pagine HTML e CSS, ma può essere limitato per mettere molte immagini sul tuo sito. In questo caso, ti consigliamo di optare per un'[offerta di hosting Web OVHcloud](https://www.ovhcloud.com/it/web-hosting/) che permetta di usufruire di uno storage di maggiore entità.
+
+Il tag HTML utilizzato per visualizzare un'immagine è il tag `<img>`. Contrariamente alle luci che abbiamo visto prima, non c'è apertura e chiusura di questo elemento. Parleremo di targhette autofermanti. Sono gli attributi di questo tag che permettono di collegare la posizione del file e il testo descrittivo dell'immagine.
+
+##### **Ottimizza le tue immagini**
+
+Un'immagine grande è un'immagine che richiederà tempo per essere caricata dal tuo browser, soprattutto se i tuoi visitatori utilizzano uno smartphone o un tablet collegato alla rete 4 o 5G.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.it-it.md]:
```suggestion
Un'immagine grande è un'immagine che richiederà tempo per essere caricata dal tuo browser, soprattutto se i tuoi visitatori utilizzano uno smartphone o un tablet collegato alla rete 4G o 5G.
```

File: pages/web/hosting/create_your_personal_webpage/guide.it-it.md [hunk 11]
```diff
@@ -0,0 +1,449 @@
...
+![Posizionamento file CSS](images/create_your_personal_webpage_4.png){.thumbnail}
+
+Ora dobbiamo collegare questo file CSS, che abbiamo chiamato per convenzione *style.css*, alla nostra pagina HTML. Questo link si effettua aggiungendo un tag `<link>` nel tag `<head>` nel file index.html:
+
+```html
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ma page personnelle</title>
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.it-it.md]: should be translated

File: pages/web/hosting/create_your_personal_webpage/guide.it-it.md [hunk 12]
```diff
@@ -0,0 +1,449 @@
...
+- [https://bootstrapthemes.co/](https://bootstrapthemes.co/).
+
+## Per saperne di più
+
+Troverai molte risorse sul Web per imparare e migliorare la tua pratica, per copiare elementi interi di codice o porzioni di codice, per aggiungere funzionalità al tuo sito senza rischiare di avere errori o malfunzionamenti. Ecco alcuni siti di riferimento:
+
+- [Iniziare con l'HTML](https://developer.mozilla.org/it/docs/Learn/HTML/Introduction_to_HTML/Getting_started)
+- [Guide di riferimento sulle luci HTML](https://developer.mozilla.org/it/docs/Web/HTML)
+- [Tutorial W3Schools su HTML](https://www.w3schools.com/html/)
+- [Tutoriels CSS Mozilla](https://developer.mozilla.org/it/docs/Web/CSS/Tutorials)
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.it-it.md]:
```suggestion
- [Tutorial CSS Mozilla](https://developer.mozilla.org/it/docs/Web/CSS/Tutorials)
```

File: pages/web/hosting/create_your_personal_webpage/guide.pl-pl.md [hunk 13]
```diff
@@ -0,0 +1,449 @@
...
+Jeśli nie znasz kodu w różnych językach, możesz skopiować/wkleić przykłady kodu, które są zawarte w tym przewodniku, pozwolą Ci one na utworzenie strony WWW, która będzie mogła być użyta na Twoim hostingu.
+
+### Jakie narzędzia?
+
+Aby utworzyć stronę www, zapisz w pliku kod źródłowy w jednym z trzech języków wymienionych powyżej. Oto ich główne nazwy rozszerzeń: *".html"* (dla plików HTML), *".css"* (dla plików CSS), *".js"* (dla plików JavaScript).
+
+Pliki można zapisać w edytorach tekstu, w tym pliki, które są domyślnie dostępne w systemie operacyjnym (notatnik, TextEdit). Darmowe rozwiązania Open Source oferują wiele dodatkowych funkcji: [Notepad++](https://notepad-plus-plus.org/), [Brackets](https://brackets.io/), [Sublime Text](https://www.sublimetext.com/) lub [micro](https://micro-editor.github.io/). Można również korzystać z IDE (Integrated Development Environment, zintegrowane środowisko rozwoju), takich jak [Visual Studio Code](https://code.visualstudio.com/) lub [Geany](https://www.geany.org/).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.pl-pl.md]:
```suggestion
Pliki można zapisać w edytorach tekstu, w tym pliki, które są domyślnie dostępne w systemie operacyjnym (notatnik, TextEdit). Darmowe rozwiązania Open Source oferują wiele dodatkowych funkcji: [Notepad++](https://notepad-plus-plus.org/), [Brackets](https://brackets.io/), [Sublime Text](https://www.sublimetext.com/) lub [Micro](https://micro-editor.github.io/). Można również korzystać z IDE (Integrated Development Environment, zintegrowane środowisko rozwoju), takich jak [Visual Studio Code](https://code.visualstudio.com/) lub [Geany](https://www.geany.org/).
```

File: pages/web/hosting/create_your_personal_webpage/guide.pl-pl.md [hunk 14]
```diff
@@ -0,0 +1,449 @@
...
+
+#### Dodaj obrazy, aby Twoja strona była bardziej atrakcyjna
+
+Internet to przede wszystkim medium wizualne. W tej części zobaczymy, jak wstawić obrazy do treści. Oferta Start 10M oferuje przestrzeń dyskową 10 MB. To wystarczy dla Twoich stron HTML i CSS, ale może być to ograniczone, jeśli chcesz umieścić dużo obrazów na swojej stronie. W tym przypadku zalecamy wybranie [oferty hostingu OVHcloud](https://www.ovhcloud.com/pl/web-hosting/) umożliwiającej szybsze przechowywanie danych.
+
+Znacznik HTML używany do wyświetlania obrazu jest znacznikiem `<img>`. W przeciwieństwie do tych, które widzieliśmy wcześniej, nie ma otwartości i zamknięcia tego elementu. Porozmawiamy o autofermantycznym tagu. Są to atrybuty tego znacznika, które umożliwiają powiązanie lokalizacji pliku z opisowym tekstem obrazu.
+
+##### **Optymalizacja obrazów**
+
+Duży obraz to obraz, który zajmie dużo czasu, zwłaszcza jeśli internauci używają smartfona lub tabletu podłączonego do sieci 4 lub 5G.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.pl-pl.md]:
```suggestion
Duży obraz to obraz, który zajmie dużo czasu, zwłaszcza jeśli internauci używają smartfona lub tabletu podłączonego do sieci 4G lub 5G.
```

File: pages/web/hosting/create_your_personal_webpage/guide.pt-pt.md [hunk 15]
```diff
@@ -0,0 +1,449 @@
...
+Se não está familiarizado com o código nas diferentes linguagens citadas, pode copiar/colar os exemplos de código que são fornecidos neste guia, que lhe permitirão ter um website explorável no seu alojamento.
+
+### Que ferramentas utilizar?
+
+Para criar uma página web, comece por escrever num ficheiro o seu código de origem a partir de uma das três linguagens acima mencionadas. Veja os principais nomes de extensão: *".html"* (para os seus ficheiros HTML), *".css"* (para os seus ficheiros CSS), *".js"* (para os seus ficheiros JavaScript).
+
+Os ficheiros podem ser escritos em editores de texto, incluindo os que estão disponíveis por predefinição no seu sistema operativo (bloco de notas, TextEdit). Numerosas soluções Open Source gratuitas propõem funcionalidades suplementares: [Notepad++](https://notepad-plus-plus.org/), [Brackets](https://brackets.io/), [Sublime Text](https://www.sublimetext.com/) ou ainda [microfone](https://micro-editor.github.io/). Também é possível utilizar um IDE (Integrated Development Environment, ambiente de desenvolvimento integrado) como o [Visual Studio Code](https://code.visualstudio.com/) ou [Geany](https://www.geany.org/).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.pt-pt.md]:
```suggestion
Os ficheiros podem ser escritos em editores de texto, incluindo os que estão disponíveis por predefinição no seu sistema operativo (bloco de notas, TextEdit). Numerosas soluções Open Source gratuitas propõem funcionalidades suplementares: [Notepad++](https://notepad-plus-plus.org/), [Brackets](https://brackets.io/), [Sublime Text](https://www.sublimetext.com/) ou ainda [Micro](https://micro-editor.github.io/). Também é possível utilizar um IDE (Integrated Development Environment, ambiente de desenvolvimento integrado) como o [Visual Studio Code](https://code.visualstudio.com/) ou [Geany](https://www.geany.org/).
```

File: pages/web/hosting/create_your_personal_webpage/guide.pt-pt.md [hunk 16]
```diff
@@ -0,0 +1,449 @@
...
+- [Guia de referência sobre as balizas HTML](https://developer.mozilla.org/pt-BR/docs/Web/HTML)
+- [Tutorial W3Schools no HTML](https://www.w3schools.com/html/)
+- [Tutoriais CSS Mozilla](https://developer.mozilla.org/pt-BR/docs/Web/CSS/Tutorials)
+- [CSS Tutorial W3 Schools](https://www.w3schools.com/css/).
+
+### Reiniciar as suas imagens
+
+Muitas ferramentas gratuitas permitem-lhe retrabalhar as suas ilustrações:
+
+- A aplicação [Fotos](https://apps.microsoft.com/store/detail/photos-microsoft/9WZDNCRFJBH4) Windows 10 e 11
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.pt-pt.md]:
```suggestion
- A aplicação [Photos](https://apps.microsoft.com/store/detail/photos-microsoft/9WZDNCRFJBH4) Windows 10 e 11
```

File: pages/web/hosting/create_your_personal_webpage/guide.pt-pt.md [hunk 17]
```diff
@@ -0,0 +1,449 @@
...
+- A aplicação [Fotos](https://support.apple.com/pt-pt/guide/photos/welcome/mac) macOS
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/create_your_personal_webpage/guide.pt-pt.md]:
```suggestion
- A aplicação [Photos](https://support.apple.com/pt-pt/guide/photos/welcome/mac) macOS
```

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**